Use kyma upgrade command in the docs (#9794)
kyma-project/kyma: docs/kyma/04-08-update.md

- If you update an existing component, make all required changes to the Helm charts of the component located in the [`resources`](https://github.com/kyma-project/kyma/tree/master/resources) directory.

> - If you add a new component to your Kyma deployment, add a top-level Helm chart for that component. Additionally, download the current [Installation custom resource](#custom-resource-installation) on the cluster and add the new component to the components list:

Reviewer: ```suggestion
- If you add a new component to your Kyma deployment, add a top-level Helm chart for that component. Additionally, download the current [Installation custom resource](#custom-resource-installation) to the cluster and add the new component to the components list:
```

Reviewer: I would go along the [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/d/download) that says: "the thing the subject is downloaded `to`".

Author: ah but then I'd use, download `from` the cluster.

   ```bash
   kubectl -n default get installation kyma-installation -o yaml > installation.yaml
   ```

- If you introduce changes in the overrides, create a file with your changes as ConfigMaps or Secrets. See the [configuration document](#configuration-helm-overrides-for-kyma-installation) for more information on overrides.

## Perform the update

> If your changes involve any modifications in the `/resources` folder that includes component chart configurations, perform the steps on "Update with resources modifications". If you only modify installation artifacts, for example by adding or removing components in the installation files or adding or removing overrides in the configuration files, perform the steps on "Update without resources modifications".

Reviewer: ```suggestion
If your changes involve any modifications in the `/resources` folder that includes component chart configurations, perform the steps under the **Update with resources modifications** tab. If you only modify installation artifacts, for example by adding or removing components in the installation files or adding or removing overrides in the configuration files, perform the steps under the **Update without resources modifications** tab.
```

Read about each update step in the following sections.

<div tabs name="perform-the-update">
   <details>

>    <summary label="update-with-resources-moditifations">

Reviewer: ```suggestion
   <summary label="update-with-resources-modifications">
```

   Update with resources modifications
   </summary>

   1. Check which version you're currently running. Run this command:

      ```bash
      kyma version
      ```

>    2. Provide the same version of the current cluster to the upgrade command. Provide also an image name and a tag so that Kyma CLI will build a Docker image with your local changes and push it to the registry. It will also trigger the update process. If you have changes for the overrides or the components list, you can also pass them using `-o` and `-c` flags.

Reviewer: ```suggestion
   2. Provide the same version of the current cluster to the upgrade command. Provide also an image name and a tag so that Kyma CLI will build a Docker image with your local changes and push it to the registry. It will also trigger the update process. If you have changes for the overrides or the components list, you can also pass them using the `-o` and `-c` flags.
```

      ```bash
      kyma upgrade -s {VERSION} --source-image {IMAGE_NAME}:{IMAGE_TAG}
      ```

   </details>
   <details>

>    <summary label="update-without-resources-moditifations">

Reviewer: ```suggestion
   <summary label="update-without-resources-modifications">
```

   Update without resources modifications
   </summary>

   1. Check which version you're currently running. Run this command:

      ```bash
      kyma version
      ```

>    2. Provide the same version of the current cluster to the upgrade command. Pass the path of the overrides file via `-o` flag and/or the path of the installation file via `-c` flag:

Reviewer: ```suggestion
   2. Provide the same version of the current cluster to the upgrade command. Pass the path of the overrides file using the `-o` flag and/or the path of the installation file using the `-c` flag:
```
